.github/workflows/ci.yml

```
@@ -54,10 +80,17 @@ jobs:
           go-version: '1.24'
 
       - name: Run golangci-lint
-        uses: golangci/golangci-lint-action@v6
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2046:warning:1:101: Quote this to prevent word splitting

(shellcheck)